src/wp-includes/general-template.php

```diff
@@ -5239,149 +5239,3 @@ function wp_heartbeat_settings( $settings ) {
-	 */
-	$login_header_url = apply_filters( 'login_headerurl', $login_header_url );
-
-	/**
-	 * Filters the title attribute of the header logo above login form.
-	 *
-	 * @since WP-2.1.0
-	 *
-	 * @param string $login_header_title Login header logo title attribute.
-	 */
-	$login_header_title = apply_filters( 'login_headertitle', $login_header_title );
-
-	if ( ! empty( $login_custom_image_html ) ) {
-		// Generate the custom login image HTML.
-		$login_image_class = 'custom-login-image-container';
-		if ( $login_custom_image_state === 2 ) {
-			$login_image_class .= ' banner';
-		}
-		$login_image_html  = sprintf(
-			'<a href="%s" class="%s" rel="home" itemprop="url" title="%s">%s</a>',
-			esc_url( $login_header_url ),
-			$login_image_class,
-			esc_attr( $login_header_title ),
-			$login_custom_image_html
-		);
-		$login_header_text = null;
-
-	} else {
-		/**
-		 * Otherwise use the ClassicPress logo.
-		 *
-		 * Set the link text (displayed only for screen readers).
-		 *
-		 * To match the URL/title set above, Multisite sites have the blog
-		 * name, while single sites get the header title.
-		 */
-		if ( is_multisite() ) {
-			$login_header_text = get_bloginfo( 'name', 'display' );
-		} else {
-			$login_header_text = $login_header_title;
-		}
-
-		$login_image_html = (
-			'<h1>'
-			. '<a href="' . esc_url( $login_header_url ) . '"'
-			. ' title="' . esc_attr( $login_header_title ) . '"'
-			. ' tabindex="-1">'
-			. $login_header_text
-			. '</a>'
-			. '</h1>'
-		);
-
-	}
-
-	/**
-	 * Filters the HTML for the image displayed on the login page.
-	 *
-	 * @since 1.1.0
-	 *
-	 * @param string $html The proposed login image HTML.
-	 * @param array  $args {
-	 *     Other relevant arguments (read-only, you must use earlier filters
-	 *     such as 'login_headerurl' to modify these values).
-	 *     @type int $login_custom_image_state
-	 *                                      0 for disabled, 1 for logo
-	 *                                      style, 2 for banner style.
-	 *     @type int|null $login_custom_image_id
-	 *                                      The image ID to use as the custom
-	 *                                      login image (or null if disabled).
-	 *     @type string $login_header_url   The URL used as a link for the
-	 *                                      login page.
-	 *     @type string $login_header_title The title attribute for the login
-	 *                                      page image.
-	 *     @type string $login_header_text  The screen reader text for the login
-	 *                                      page image.
-	 * }
-	 */
-	return apply_filters(
-		'login_image_html',
-		$login_image_html,
-		array(
-			'login_custom_image_state' => $login_custom_image_state,
-			'login_custom_image_id'    => $login_custom_image_id,
-			'login_header_url'         => $login_header_url,
-			'login_header_title'       => $login_header_title,
-			'login_header_text'        => $login_header_text,
-		)
-	);
-}
```

Comment:
Is it right to remove custom login image?

Reply:
Good spot - thank you, will add a further commit.